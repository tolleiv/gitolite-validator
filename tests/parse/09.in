# sample conf/gitolite.conf file

@staff              =   dilbert alice           # groups
@projects           =   foo bar

repo @projects baz                              # repos
    RW+             =   @staff                  # rules
    -       master  =   ashok                   # with
    RW              =   ashok                   # comments and comments
    R               =   wally
    option deny-rules           =   1           # options
    config hooks.emailprefix    = '[%GL_REPO] ' # git-config
