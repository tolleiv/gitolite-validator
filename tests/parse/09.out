root:
  group:
    name: staff
    members:
      dilbert
      alice
  group:
    name: projects
    members:
      foo
      bar
  repo:
    paths:
      @projects
      baz
    rules:
      rule:
        permission: RW+
        members:
          @staff
      rule:
        permission: -
        refex: master
        members:
          ashok
      rule:
        permission: RW
        members:
          ashok
      rule:
        permission: R
        members:
          wally
    configs:
        hooks.emailprefix = '[%GL_REPO]'
    options:
        deny-rules = 1
